support-e2e/tests/utils/testUserDetails.ts

```diff
@@ -1,7 +1,48 @@
 import { Page } from '@playwright/test';
 import { TestFields } from './userFields';
 
-export const setTestUserDetails = async (
+const selectDeliveryAgent = async (page: Page) => {
+	// Depending on whether there are one or multiple delivery agents we need to do different things here.
+	// If there are multiple delivery agents, we need to select one of them, if there is only one we do not.
+	const deliveryAgentLabel = page.locator(
+		'div:text-matches("Delivery provider|Select delivery provider")', // This will match both labels
+	);
+	await deliveryAgentLabel.waitFor({ state: 'visible' });
+
+	const deliveryAgentRadioButton = page.locator(
+		'fieldSet#delivery-provider input[type="radio"]',
+	);
+
+	if ((await deliveryAgentRadioButton.count()) > 0) {
+		// If there are multiple delivery agents, select the first one
+		await deliveryAgentRadioButton.first().check();
+	}
+};
+
+const requireAddress = (
```

Comment:
Could we derive this from the fields which are presented to the user? I.e. if address fields are rendered then fill them in? Just thinking this might prevent us from having this logic in two places? (The implementation and the tests?) Maybe this could be a future optimisation.